file_and_modules.md

```diff
@@ -41,7 +41,35 @@
 * We can import a module using import keyword.
 * If it is not available it will give import error.
 * dir() method tells what these modules provides.
+* What's [The Zen of Python] (https://www.python.org/dev/peps/pep-0020/) ? 
 ```
+ # python
+Python 3.6.8 (default, Jan 31 2019, 09:38:34) 
+[GCC 8.2.1 20181215 (Red Hat 8.2.1-6)] on linux
+Type "help", "copyright", "credits" or "license" for more information.
```

Comment:
Remove this and just use python interpreter, otherwise it will confuse user to use older version.

Reply:
done!